sql/plan/drop_role.go

```diff
@@ -19,23 +19,27 @@ import (
 	"strings"
 
 	"github.com/dolthub/go-mysql-server/sql"
+	"github.com/dolthub/go-mysql-server/sql/grant_tables"
 )
 
 // DropRole represents the statement DROP ROLE.
 type DropRole struct {
-	IfExists bool
-	Roles    []UserName
+	IfExists    bool
+	Roles       []UserName
+	GrantTables sql.Database
 }
 
 // NewDropRole returns a new DropRole node.
 func NewDropRole(ifExists bool, roles []UserName) *DropRole {
 	return &DropRole{
-		IfExists: ifExists,
-		Roles:    roles,
+		IfExists:    ifExists,
+		Roles:       roles,
+		GrantTables: sql.UnresolvedDatabase("mysql"),
```

Comment:
There's are missing pieces here that I can't see for how an integrator persists these grant tables after the session end. Would be nice to see them sketched out soon.

Reply:
My plan was to pass the entries themselves to integrators. So to persist the contents of the`role_edges` table, I’d just pass in `*RoleEdge`. Same for `user`, just pass `*User`.